lib/node_modules/@stdlib/math/base/special/besselj0/lib/besselj0.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+/*
+* The original C++ code and copyright notice is from the [Boost library]{@link https://github.com/boostorg/math/blob/develop/include/boost/math/special_functions/detail/bessel_j0.hpp}.
+*
+* The implementation has been modified for JavaScript.
+*/
+
+/*
+* Copyright John Maddock 2006-7, 2013-14.
+* Copyright Paul A. Bristow 2007, 2013-14.
+* Copyright Nikhar Agrawal 2013-14
+* Copyright Christopher Kormanyos 2013-14
+* Use, modification and distribution are subject to the
+* Boost Software License, Version 1.0. (See accompanying file
+* LICENSE or copy at http://www.boost.org/LICENSE_1_0.txt)
+*/
+
+// MODULES //
+
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var sincos = require( '@stdlib/math/base/special/sincos' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var evaluateRational = require( './evaluate_rational.js' );
```

Comment:
Why do you need this function rather than `@stdlib/math/base/tools/evalrational`?

Reply:
Ah, good catch. I don't. I thought maybe @Planeshifter had pulled it in, but I searched for `evaluate` instead of `eval`. 👍

Reply:
Yeah, one of the things on our todo is to add search to the docs. @Planeshifter prototyped something a while back ago, but we have not gotten it ready for prod.